source/gui/nvdaControls.py

```diff
@@ -414,29 +358,24 @@ def __init__(
 		if helpId is not None:
 			self.helpId = helpId
 		super().__init__(parent, title, message, dialogType)
+		if helpId is not None:
+			# Help event has already been bound (in supersuperclass), so we need to re-bind it.
+			self.bindHelpEvent(helpId, self)
```

Comment:
this is a little concerning, doesn't that mean add-ons using the old `MessageDialog` need to do this otherwise face breaking changes? can this be done from the old `MessageDialog`?